### PR TITLE
Handle scalars in count_params

### PR DESCRIPTION
**Pull request by instagibbs**
Simply returns 1 for '()' shapes.

### CODE REVIEW

File: lasagne/layers/helper.py [hunk 1]
```diff
@@ -251,7 +251,7 @@ def count_params(layer):
     """
     params = get_all_params(layer)
     shapes = [p.get_value().shape for p in params]
-    counts = [np.prod(shape) for shape in shapes]
+    counts = [np.prod(shape) if shape != () else 1 for shape in shapes]
```
Comment (benanne) [inline lasagne/layers/helper.py]: Are you sure this is necessary? I get:
```python
>>> import numpy as np
>>> np.prod(())
1.0
```

Comment (instagibbs) [inline lasagne/layers/helper.py]: You're right. On that note it brings up another question. 

A blank numpy array is also computed as 1.

Comment (benanne) [inline lasagne/layers/helper.py]: That's the behaviour I would expect though. What question does it bring up? :)

Comment (instagibbs) [inline lasagne/layers/helper.py]: 0 parameters being counted as 1?

Comment (benanne) [inline lasagne/layers/helper.py]: No, it's the shape that has 0 elements, not the parameter array itself.

Comment (instagibbs) [inline lasagne/layers/helper.py]: You know what? I need more coffee. I was writing the wrong thing in my command line.

Comment (benanne) [inline lasagne/layers/helper.py]: Haha, don't we all :)

**Pull request closed without merging**